### ISSUE TITLE
Secret Zelda Levels

### ISSUE THREAD

**Issue by grady404** [cooper-anderson/Blockade#46]
So first off, I think there should be an entire menu or section of the game specifically for secret/bonus levels. I'd like to see this game have a ton of secret content, as it would definitely make the experience a lot more enjoyable and interesting for players. So the first bonus level you would unlock (quite early in the game, maybe after beating around 4-8 levels) would be a room with no goal pads, only 8 weird switches with music notes on them. You wouldn't quite know what they did at first, but after causing one of the blocks to come to rest on one of them, you'd hear a musical note, and you'd eventually discover that each of the eight pads played a different note. Those notes would be G#, A, D#, E, F#, G, G#, and C, in ascending order. Any guess what the objective might be?

https://www.youtube.com/watch?v=9d3qCPcMgH4

If you play the notes in the right order, the game will play the chime back for you (so that you know it did something), and the first Zelda level would unlock. The level select for these four levels would be in the shape of a Triforce, with level 1 occupying the top triangle, level 2 on the bottom left, level 3 on the bottom right, and level 4 appearing in the center of the Triforce after you complete the other three.

**Comment by cooperuser** [comment]
Interesting idea, when you said that there should be a whole different section of the game, that reminded me of the star world from Mario. But I do like this idea

**Comment by grady404** [comment]
Well the thing is, these Zelda levels are easier, so I think they should unlock earlier in the game, and it should be more of a thing for people who actually make the connection and realize that this game is reminiscent of the ice puzzles from Twilight Princess (then they will see this level and immediately have the idea of the Zelda chime)

**Comment by cooperuser** [comment]
yeah

**Comment by cooperuser** [comment]
the thing is there are 8 chimes, and only 7 notes

**Comment by cooperuser** [comment]
also im not sure if this was your intention, but we could have these "note-blocks" placed semi-randomly in a grid and make you do a puzzle to get it right

**Comment by grady404** [comment]
Nope, I put eight there

**Comment by cooperuser** [comment]
G# twice

**Comment by grady404** [comment]
And yeah, that was the intention

**Comment by grady404** [comment]
No, there's G and G#

**Comment by grady404** [comment]
Another cool aspect of the puzzle would be that you would have to _avoid_ triggering the plates that weren't next in the sequence

**Comment by cooperuser** [comment]
there are 3 G#s played in the tune, two on the higher octave and one on the lower

**Comment by grady404** [comment]
Well true, but they are different tones

**Comment by cooperuser** [comment]
in the two there are 7 different notes

**Comment by cooperuser** [comment]
<img width="430" alt="screen shot 2016-09-11 at 1 53 12 am" src="https://cloud.githubusercontent.com/assets/21961037/18416228/9ac35c5a-77c2-11e6-8150-c28f2881648d.png">

**Comment by cooperuser** [comment]
it could look something like that, where you have to set it up

**Comment by cooperuser** [comment]
but i guess that would be a bit hard

**Comment by grady404** [comment]
Yeah something like that but maybe the grid is larger so it's not so tight

**Comment by cooperuser** [comment]
yeah
